views/question-creation.pug

```diff
@@ -1,5 +1,10 @@
 form.form-user#questionform
   p#result
+
+  .form-group
+    label(for='show') Show?
+    input.form-control#show(type='checkbox' name='show' checked=false value=false)
```

Comment:
is there a difference between checked and value? why do we need to set both?

Reply:
Don't think we need value anymore as I was using it previsouly. I'll remove it and check just to be sure